ndbench-cass-plugins/src/main/java/com/netflix/ndbench/plugin/cass/CassJavaDriverBatch.java

```diff
@@ -0,0 +1,146 @@
+public class CassJavaDriverBatch extends CJavaDriverBasePlugin {
+
+    //Settings
+    private static String TableName2;
+    private static Integer batchSize;
+    private static Boolean useTimeStamp;
+    private static Boolean useMultiPartition;
+
+    Random randomObj = new Random();
+
+    protected PreparedStatement writePstmt2;
+
+    @Inject
+    public CassJavaDriverBatch(PropertyFactory propertyFactory, CassJavaDriverManager javaDriverManager) {
+        super(propertyFactory, javaDriverManager);
+    }
+
+    @Override
+    void prepStatements(Session session) {
+
+        readPstmt = session.prepare(" SELECT cyclist_name, expense_id, amount, description, paid FROM "+TableName+" WHERE cyclist_name = ?" );
+
+        writePstmt = session.prepare("INSERT INTO "+TableName+" (cyclist_name, expense_id, amount, description, paid) VALUES (?, ?, ?, ?, ?)");
+        writePstmt2 = session.prepare("INSERT INTO "+TableName2+" (expense_id, cyclist_name) VALUES (?, ?)");
+
+
+    }
+
+    @Override
+    void upsertKeyspace(Session session) {
+        upsertGenereicKeyspace();
+    }
+
+    @Override
+    void upsertCF(Session session) {
+        session.execute("CREATE TABLE IF NOT EXISTS "+TableName+" (cyclist_name text, balance float STATIC, expense_id int, amount float, description text, paid boolean, PRIMARY KEY (cyclist_name, expense_id) )");
+        session.execute("CREATE TABLE IF NOT EXISTS "+TableName2+" (expense_id int, cyclist_name text, PRIMARY KEY (expense_id, cyclist_name)) ");
+    }
+
+    @Override
+    void preInit() {
+        TableName2  = propertyFactory.getProperty(NdBenchConstants.PROP_PREFIX+"cass.cfname2").asString("test2").get();
+        batchSize = propertyFactory.getProperty(NdBenchConstants.PROP_PREFIX+"cass.batchSize").asInteger(3).get();
+        useTimeStamp = propertyFactory.getProperty(NdBenchConstants.PROP_PREFIX+"cass.useTimestamp").asBoolean(true).get();
+        useMultiPartition = propertyFactory.getProperty(NdBenchConstants.PROP_PREFIX+"cass.useMultiPartition").asBoolean(false).get();
+    }
+
+    @Override
+    void postInit() {
+
+    }
+
+    @Override
+    public String readSingle(String key) throws Exception {
+        BoundStatement statement = readPstmt.bind();
+        statement.setString("cyclist_name", key);
+        statement.setConsistencyLevel(this.ReadConsistencyLevel);
+        ResultSet rs = session.execute(statement);
+
+        List<Row> result = rs.all();
+
+        if (!result.isEmpty()) {
+
+            if (result.size() < 1) {
+                throw new Exception("Expecting non zero rows for cyclist_name: " + key);
+            }
+        } else {
+            return CacheMiss;
+        }
+
+        return ResultOK;
+    }
+
+    @Override
+    public String writeSingle(String key) throws Exception {
+
+        BatchStatement batch = new BatchStatement();
+        for (int i = 0; i < this.batchSize; i++) {
+
+            BoundStatement bStmt;
+            if(useMultiPartition)
```

Comment:
you can decrease the if...else number of statements and lines of code by having one if and then else `bStmt = getBStmtTable1(key);`